etc/conda3_packages-linux-64.yml

```diff
@@ -1,161 +1,254 @@
+  - gcc_linux-64=7.3.0=h553295d_18
+  - gettext=0.19.8.1=hc5be6a0_1002
+  - gflags=2.2.2=he1b5a44_1002
+  - gfortran_impl_linux-64=7.3.0=hdf63c60_5
+  - gfortran_linux-64=7.3.0=h553295d_18
+  - git-lfs=2.10.0=0
+  - glib=2.64.2=h6f030ca_0
+  - glog=0.4.0=h49b9bf7_3
+  - grpc-cpp=1.25.0=h213be95_2
+  - gsl=2.6=h294904e_0
+  - gst-plugins-base=1.14.5=h0935bb2_2
+  - gstreamer=1.14.5=h36ae1b5_2
+  - gxx_impl_linux-64=7.3.0=hdf63c60_5
+  - gxx_linux-64=7.3.0=h553295d_18
+  - h5py=2.10.0=nompi_py37h513d04c_102
+  - hdf5=1.10.5=nompi_h3c11f04_1104
+  - healpy=1.13.0=py37hf476568_1
+  - icu=64.2=he1b5a44_1
+  - idna=2.8=py37_1000
+  - imageio=2.8.0=py_0
+  - importlib-metadata=1.6.0=py37hc8dfbb8_0
+  - importlib_metadata=1.6.0=0
+  - itsdangerous=1.1.0=py_0
+  - jinja2=2.11.2=pyh9f0ad1d_0
+  - jmespath=0.9.5=py_0
   - joblib=0.14.1=py_0
-  - jpeg=9b=h024ee3a_2
-  - jsondiff=1.1.1=py37_0
-  - jsonpickle=1.2=py_0
-  - kiwisolver=1.1.0=py37he6710b0_0
-  - ld_impl_linux-64=2.33.1=h53a641e_7
-  - libboost=1.71.0=h97c9712_0
-  - libedit=3.1.20181209=hc058e9b_0
-  - libevent=2.1.8=h1ba5d50_0
-  - libffi=3.2.1=hd88cf55_4
-  - libgcc-ng=9.1.0=hdf63c60_0
-  - libgfortran-ng=7.3.0=hdf63c60_0
-  - libopenblas=0.3.6=h5a2b251_2
-  - libpng=1.6.37=hbc83047_0
-  - libprotobuf=3.11.2=hd408876_0
-  - libstdcxx-ng=9.1.0=hdf63c60_0
-  - libuuid=1.0.3=h1bed415_2
-  - libxcb=1.13=h1bed415_1
-  - libxml2=2.9.9=hea5a465_1
-  - lz4-c=1.8.1.2=h14c3975_0
-  - lzo=2.10=h49e0be7_2
-  - markupsafe=1.1.1=py37h7b6447c_0
-  - matplotlib=3.0.3=py37h5429711_0
-  - mock=4.0.1=py_0
+  - jpeg=9c=h14c3975_1001
+  - jsondiff=1.1.2=py_0
+  - jsonpatch=1.24=py_0
+  - jsonpickle=1.4.1=pyh9f0ad1d_0
+  - jsonpointer=2.0=py_0
+  - jsonschema=3.2.0=py37hc8dfbb8_1
+  - kiwisolver=1.2.0=py37h99015e2_0
+  - krb5=1.17.1=h2fd8d38_0
+  - lapack=3.6.1=ha44fe06_2
+  - ld_impl_linux-64=2.34=h53a641e_0
+  - libapr=1.6.5=h516909a_2
+  - libapriconv=1.2.2=h516909a_2
+  - libaprutil=1.6.1=ha1d75be_2
+  - libblas=3.8.0=16_openblas
+  - libcblas=3.8.0=16_openblas
+  - libcurl=7.69.1=hf7181ac_0
+  - libedit=3.1.20170329=hf8c457e_1001
+  - libevent=2.1.10=h72c5cf5_0
+  - libffi=3.2.1=he1b5a44_1007
+  - libgcc-ng=9.2.0=h24d8f2e_2
+  - libgfortran=3.0.0=1
+  - libgfortran-ng=7.3.0=hdf63c60_5
+  - libgomp=9.2.0=h24d8f2e_2
+  - libiconv=1.15=h516909a_1006
+  - liblapack=3.8.0=16_openblas
+  - libopenblas=0.3.9=h5ec1e0e_0
+  - libpng=1.6.37=hed695b0_1
+  - libprotobuf=3.11.0=h8b12597_0
+  - libssh2=1.8.2=h22169c7_2
+  - libstdcxx-ng=9.2.0=hdf63c60_2
+  - libtiff=4.1.0=hc7e4089_6
+  - libuuid=2.32.1=h14c3975_1000
+  - libuv=1.34.0=h516909a_0
+  - libwebp-base=1.1.0=h516909a_3
+  - libxcb=1.13=h14c3975_1002
+  - libxml2=2.9.10=hee79883_0
+  - llvm-openmp=10.0.0=hc9558a2_0
+  - lmfit=1.0.0=py_0
+  - log4cxx=0.10.0=h66fa5af_2
+  - lsstdesc.coord=1.2.1=py37h99015e2_0
+  - lz4-c=1.8.3=he1b5a44_1001
+  - lzo=2.10=h14c3975_1000
+  - make=4.3=h516909a_0
+  - markupsafe=1.1.1=py37h8f50634_1
+  - matplotlib=3.2.1=0
```

Comment:
Does matplotlib now work then?  Last time I tried building with a new matplotlib everything broke because of matplotlib/matplotlib#15410. (see https://jira.lsstcorp.org/browse/DM-22818).

Reply:
I assume so - this build was done with this package list, but a different sha because it was before I reorganized the commits:
https://ci.lsst.codes/job/stack-os-matrix/31695
https://ci.lsst.codes/job/stack-os-matrix/31695/parameters/

Possibly this was related?
https://github.com/conda-forge/matplotlib-feedstock/issues/236